src/main/java/com/google/api/codegen/transformer/ruby/RubySurfaceNamer.java

```diff
@@ -98,4 +99,11 @@ public String getServiceFileImportFromService(Interface service) {
   public String getProtoFileImportFromService(Interface service) {
     return service.getFile().getSimpleName().replace(".proto", "_pb");
   }
+
+  // For some reason this is only being called for tests.
+  @Override
+  public String getSourceFilePath(String path, String className) {
+    // Class name is some Java thing like LanguageServiceTest
+    return path + File.separator + className + "_test.jorge.rb";
```

Comment:
When I started my career, I worked on communication satellites. The techs always hung big red flags from anything attached to the satellite that should not be there on launch day. My cyber-equivalent for coding is to put a FIXME comment next to anything I want to purge before merge. As others have pointed out to me, FIXME comments do not conform to the Google standard, which is exactly the point: they're easy to find in a `git grep`, and they should be removed (along with the offending code) before code review is complete.